core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -310,26 +309,26 @@ browse from the page internal links
      * @memberof DotEditContentComponent
      */
     showFavoritePageDialog(openDialog: boolean): void {
-        this.pageState$.pipe(take(1)).subscribe((pageState: DotPageRenderState) => {
-            if (openDialog) {
-                this.dialogService.open(DotFavoritePageComponent, {
-                    header: this.dotMessageService.get('favoritePage.dialog.header.add.page'),
-                    width: '80rem',
-                    data: {
-                        page: {
-                            pageState: pageState,
-                            pageRenderedHtml: pageState.params.page.rendered || null
-                        },
-                        onSave: (favoritePageUrl: string) => {
-                            this.updateFavoritePageIconStatus(favoritePageUrl);
-                        },
-                        onDelete: (favoritePageUrl: string) => {
-                            this.updateFavoritePageIconStatus(favoritePageUrl);
-                        }
+        if (openDialog) {
+            const favoritePageUrl = generateDotFavoritePageUrl(this.pageStateInternal);
+
+            this.dialogService.open(DotFavoritePageComponent, {
```

Comment:
`DotFavoritePageComponent` going to need a loading skeleton (as per previous comment)

Reply:
as discussed on chat, it won't be needed!